src/Kunstmaan/NodeBundle/EventListener/NodeTranslationListener.php

```diff
@@ -324,7 +332,16 @@ private function ensureUniqueUrl(
             ).' node(s) that match url \''.$translation->getUrl().'\''
         );
 
-        if (count($translations) > 0) {
+        $translationsWithSameUrl = array();
+
+        /** @var NodeTranslation $trans */
+        foreach ($translations as $trans) {
+            if ($this->pagesConfiguration->isStructureNode($trans->getPublicNodeVersion()->getRefEntityName())) {
```

Comment:
Is this not supposed to be negatory? As I'm reading this you're only including structured nodes instead of ignoring them.